src/pixelator/utils.py

```diff
@@ -360,3 +371,60 @@ def get_process_pool_executor(ctx, **kwargs):
         max_workers=ctx.obj["CORES"],
         **kwargs,
     )
+
+
+def get_read_sample_name(read: str) -> str:
+    """Extract the sample name from a read file.
+
+    Strip fq.gz or fastq.gz extension and remove R1/R2 suffixes.
+    Supported R1 R2 identifieds are:
+
+    _R1,_R2 | _r1, _r2 | _1, _2 | .R1, .R2 | .r1, .r2
+
+    :param read: filename of a fastq read file
+    :return str: sample name
```

Comment:
return vs returns?  Are both supported in sphinx and autodoc?

Reply:
Yes, they are equivalent.